source/agora/consensus/Validation.d

```diff
@@ -468,8 +468,8 @@ unittest
         );
         secondTx.inputs[0].signature = key_pairs[0].secret.sign(hashFull(secondTx)[]);
 
-        // Second Transaction is invalid.
-        assert(!secondTx.isValid(findUTXO, 0));
+        // Second Transaction is valid.
+        assert(secondTx.isValid(findUTXO, 0));
```

Comment:
Look at the comment in line 446. That comment needs to be changed too.

Reply:
Good eye :) applied the suggest